File: check_ru.pl
use strict;
use warnings;
use utf8;
use open qw(:std :utf8);

my $param1 = @ARGV;
my $filename;

if ($param1) {
    $filename = chomp($param1);
}
else {
    $filename = 'Module.bsl';
}

open(my $fh, '<:encoding(UTF-8)', $filename)
    or die "Could not open file '$filename' $!";

my $count = 0;
my $ind = 1;
while (my $line = <$fh>) {
    ind++;
    if ($line =~ /[А-яёЁ]/g) {
        print "$ind  - $line\n";
        $count++;
    }
}

close($fh);

print ("Find $count lines with russian letters\n");

File: check_ru.py
import re
import sys

print(sys.argv)

if len(sys.argv) != 2:
    filename = "Module.bsl"
else:
    filename = sys.argv[1]

print(filename)

with open(filename, 'r', encoding="utf-8") as file:
    lines = file.readlines()

col = 0
ind = 1
for line in lines:
    ind += 1
    if not re.search('[А-яёЁ]', line):
        continue
    print(f"{ind} - {line.rstrip()}")
    col += 1

print(f"Found {col} lines")
